mirgecom/eos.py

```diff
@@ -233,3 +232,236 @@ def total_energy(self, cv, pressure):
         """
         return (pressure / (self._gamma - 1.0)
                 + self.kinetic_energy(cv))
+
+
+class PyrometheusMixture(GasEOS):
+    r"""Ideal gas mixture ($p = \rho{R}_\mathtt{mix}{T}$).
+
+    This is the *Pyrometheus*-based EOS. Please refer to the :any:`documentation of
```

Comment:
```suggestion
    This is the :mod:`pyrometheus`-based EOS. Please refer to the :any:`documentation of
```

Reply:
This suggestion and a few other minor improvements (117942b)